app/views/curation_concerns/single_use_links/new_download.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="single-use-link">
+<h1>Single Use Link</h1>
+<p>Anyone can use the following link once to download the file</p>
+<%= link_to @asset, @link, class: 'download-link' %> <%= link_to raw('<i class="glyphicon glyphicon-link large-icon"></i>'), '#', class: 'copypaste itemicon itemcode', title: 'Copy File URL', id: "copy_link_#{@asset.id}" %>
```

Comment:
The `link_to_raw` bit refers to javascript code that isn't there so that bit should be removed. The download links only work when they are opened in a new tab so I think ` target: '_blank'` should be provided as an option after `class: 'download-link'`